client/components/mma/accountoverview/InAppPurchaseCard.tsx

```diff
@@ -0,0 +1,43 @@
+import { css } from '@emotion/react';
+import { from, headline, palette, space } from '@guardian/source-foundations';
+import type { AppSubscription } from '../../../../shared/mpapiResponse';
+import { InfoSummary } from '../paymentUpdate/Summary';
+import { Card } from '../shared/Card';
+
+const productTitleCss = css`
+	${headline.xxsmall({ fontWeight: 'bold' })};
+	color: ${palette.neutral[100]};
+	margin: 0;
+	max-width: 20ch;
+
+	${from.tablet} {
+		${headline.small({ fontWeight: 'bold' })};
+	}
+`;
+
+export const InAppPurchaseCard = (props: {
+	inAppPurchase: AppSubscription;
+}) => {
+	return (
+		<>
+			{props.inAppPurchase.cancellationTimestamp && (
+				<InfoSummary
+					message="Your IAP Was cancelled TODO"
+					cssOverrides={css`
+						color: ${palette.neutral[0]};
```

Comment:
Do you need to override the colour now you've fixed my mistake in the component itself? 😀

Reply:
nope, in fact I don't think this even worked!